<launch>

    <node pkg="green_fundamentals" type="publish_map.py" name="publish_map.py" output="screen" />

    <node pkg="green_fundamentals" type="map_generator" name="map_generator" output="screen" />

    <node pkg="green_fundamentals" type="obstacle_detector" name="obstacle_detector" output="screen" />

    <node pkg="green_fundamentals" type="robot_mover" name="robot_mover" output="screen" />

    <node pkg="green_fundamentals" type="localizer" name="localizer" output="screen" />

    <node pkg="green_fundamentals" type="local_planner" name="local_planner" output="screen" />
    
</launch>
